metricflow/test/fixtures/manifest_fixtures.py

```diff
@@ -3,7 +3,8 @@
 import logging
 from dataclasses import dataclass
 from enum import Enum
-from typing import Dict, Mapping, OrderedDict, Sequence, Tuple
+from typing import Dict, Mapping, OrderedDict, Sequence
+from typing import Tuple
```

Comment:
Weird, but whatever....

Reply:
I've been using the IDE to add imports, so maybe a bug there? I'll update.